tests/unittests/helpers/testers.py

```diff
@@ -300,12 +301,13 @@ def _functional_test(
         _assert_allclose(tm_result, sk_result, atol=atol)
 
 
-def _assert_half_support(
+def _assert_dtype_support(
     metric_module: Optional[Metric],
     metric_functional: Optional[Callable],
     preds: Tensor,
     target: Tensor,
     device: str = "cpu",
+    dtype: torch.dtype = torch.half,
     **kwargs_update,
 ):
     """Test if an metric can be used with half precision tensors.
```

Comment:
typo: (not created in this PR, but I guess it's okay to fix)
```suggestion
    """Test if n metric can be used with half precision tensors.
```